ensembl/htdocs/info/genome/variation/prediction/predicted_data.html

```diff
@@ -44,6 +44,8 @@ <h1 style="margin-top:15px">Ensembl Variation - Calculated variant consequences<
 For structural variants consequence terms are calculated on the fly for display 
 on our website or API access. 
 You can use this pipeline to annotate your own data via <a href="/info/docs/tools/vep">VEP</a>. 
+By default command line VEP will include annotations of upstream/downstream within 5kb distance
```

Comment:
Is this behaviour specific to the command line? 

I suggest:
`By default, VEP will include upstream and downstream annotations for variants within 5kb of a nearby feature (e.g. upstream_gene_variant...)`